bridged-nft-token/src/internal.rs

```diff
@@ -0,0 +1,167 @@
+use crate::*;
+use near_sdk::{log, CryptoHash};
+use std::mem::size_of;
+
+pub(crate) fn hash_account_id(account_id: &AccountId) -> CryptoHash {
+    let mut hash = CryptoHash::default();
+    hash.copy_from_slice(&env::sha256(account_id.as_bytes()));
+    hash
+}
+
+pub(crate) fn assert_one_yocto() {
+    assert_eq!(
+        env::attached_deposit(),
+        1,
+        "Requires attached deposit of exactly 1 yoctoNEAR",
+    )
+}
+
+pub(crate) fn assert_at_least_one_yocto() {
+    assert!(
+        env::attached_deposit() >= 1,
+        "Requires attached deposit of at least 1 yoctoNEAR",
+    )
+}
+
+pub(crate) fn refund_deposit(storage_used: u64) {
+    let required_cost = env::storage_byte_cost() * Balance::from(storage_used);
+    let attached_deposit = env::attached_deposit();
+
+    assert!(
+        required_cost <= attached_deposit,
+        "Must attach {} yoctoNEAR to cover storage",
+        required_cost,
+    );
+
+    let refund = attached_deposit - required_cost;
+    if refund > 1 {
```

Comment:
Either use `> 0` or we can estimate the cost of scheduling a transfer, and only sending back if it is worth it.

Reply:
it was removed and replaced by the one contained inside the standard library